spec/requests/api/v1/users/sign_out_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'User sign out', type: :request do
+  let(:user) { create(:user) }
+  subject { delete destroy_api_v1_user_session_path, headers: user_tokens, as: :json }
+
+  context 'when user tokens are correct' do
+    before { subject }
+    let(:user_tokens) { user.create_new_auth_token }
+
+    it 'returns status code ok' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'returns success true in the body' do
+      expect(response.body).to eq({ success: true }.to_json)
+    end
+
+    it 'returns errors when signin out again' do
+      delete destroy_api_v1_user_session_path, headers: user_tokens, as: :json
+
+      expect(response).to have_http_status(:not_found)
+      expect(response.body).to eq({ success: false,
+                                    errors: ['User was not found or was not logged in.'] }.to_json)
+    end
```

Comment:
WDYT of doing it like this:
```suggestion
    context 'when the user is already signed out' do
      before do
        delete destroy_api_v1_user_session_path, headers: user_tokens, as: :json
      end
      
      it 'returns not found' do
        expect(response).to have_http_status(:not_found)
      end
      
      it 'returns the corresponding error' do
        expect(response.body).to eq({ success: false,
                                      errors: ['User was not found or was not logged in.'] }.to_json)
      end
    end
```
?